resources/app/log-collector.yaml

```diff
@@ -38,7 +38,7 @@ data:
         "ForwarderTmplFile": "/opt/logrange/gravity/config/forward-tmpl.json",
         "CronQueries": [
           {
-            "Query": "truncate minsize 4mb maxsize 500mb maxdbsize 10gb",
+            "Query": "truncate maxdbsize 10gb",
```

Comment:
Also, what is the reason for removing "minsize" and "maxsize" fields?

Reply:
reading the docs I get that `minsize` and `maxsize` are used to filter the logs that are considered for truncating.

So if those are not considered (cause they do not comply to the min-max size rules) they could potentially grow bigger than 10GB, as a whole.

Instead if I moved to use "truncate maxdbsize 10gb" so that no logs will be filtered out, no matter the size of the single partition.

Reply:
Yeah, it makes sense. Let's check with the Logrange team why they put it there in the first place.